src/angularjs/src/app/analysis-jobs/list/analysis-jobs-list.html

```diff
@@ -24,6 +24,7 @@ <h1 class="no-margin-top">Analysis Jobs</h1>
             <td>{{analysisJob.status | displayStatus}}</td>
             <td>{{analysisJob.neighborhood.label}}</td>
             <td>{{analysisJob.batch}}</td>
+            <td>{{analysisJob.local_upload_task.status | displayStatus}}</td>
```

Comment:
Just a note: this sort of works by coincidence--the `AnalysisUploadTask` statuses are a subset of the `AnalysisJob` statuses, but they're parallel constructs that could have been different.  If we wanted to add a status for the uploads that didn't match one already defined on jobs, we'd have to decide whether to make the filter a superset or split it.  But fortunately we don't.

Reply:
So indeed.  The capital letters drew my eye and I missed the comment.  Excellent.